ansible/roles/local_build/tasks/node.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Install nvm
+  shell: "cd; curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash"
```

Comment:
we should consider that we fork this repository/project, as this can be an attack vector. we download a shell script from 3rd party and execute it.

Reply:
Maybe we could just directly download a specific binary tarball (with a checksum) from nodejs.org. I'm not sure if node needs any configuration or if you can just unpack and then run the binaries directly.

Reply:
We could download https://github.com/nvm-sh/nvm/archive/v0.34.0.tar.gz and compare SHA hash that it is the same what it is now. so that we store the hash with the Ansible and download the full source tar ball, from which we can run the install.sh, as long as we have done a code review on the tarball.

Reply:
It seems that install.sh downloads the latest version of nvm from git so there's not much security benefit from checksumming if it just gets untrusted code from the web anyway.

Reply:
then we should use that nodejs as you suggested. it should work out of the box by just extracting if we have the platform binaries. what we might want to do is to add the bin folder to PATH, which I presume that nvm is doing as well

Reply:
ahem, lets stop right here.

Reply:
we are out of scope of this issue. the fact is that we already are having the same pattern there in another role. what we should do, instead of copying the lines, is to use that role instead. Then we can fix this issue which was spoken here with in the right scope and with different issue.